doc/tools/coqrst/coqdomain.py

```diff
@@ -986,7 +994,7 @@ def add_coq_output_1(self, repl, node):
             repl.sendone('Unset Coqtop Exit On Error.')
         if options['warn']:
             repl.sendone('Set Warnings "default".')
-        for sentence in self.split_sentences(node.rawsource):
+        for sentence in self.split_sentences(node):
```

Comment:
No need to change this argument, I think

Reply:
Done